# Publish all joint states -----------------------------------
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

# Position Controllers ---------------------------------------
joint1_position_controller:
  type: effort_controllers/JointPositionController
  joint: base_to_00
  pid: {p: 1000.00, i: 200.00, d: 200.00}

joint2_position_controller:
  type: effort_controllers/JointPositionController
  joint: 00_to_01
  pid: {p: 5000.00, i: 500.00, d: 500.00}
  
joint3_position_controller:
  type: effort_controllers/JointPositionController
  joint: 01_to_02
  pid: {p: 1000.00, i: 750.00, d: 50.00}